pages/linux/hostnamectl.md

```diff
@@ -9,3 +9,11 @@
 - Set the hostname of the computer:
 
 `sudo hostnamectl set-hostname "{{some_hostname}}"`
+
+- Set a pretty hostname for the computer:
+
+`sudo hostnamectl set-hostname --static "{{some_hostname.example.com}}" && sudo hostnamectl set-hostname --pretty "{{some_hostname}}"`
```

Comment:
I don't think underscores are actually allowed in domain names...see the [IETF DNS Specification (RFC 1035)](https://tools.ietf.org/html/rfc1035) :)
```suggestion
`sudo hostnamectl set-hostname --static "{{hostname.example.com}}" && sudo hostnamectl set-hostname --pretty "{{hostname}}"`
```